test/Interop/Cxx/value-witness-table/custom-virtual-destructors.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop -Xcc -fno-rtti)
```

Comment:
Why `-fno-rtti`? Please document here and in other tests that need it.

Reply:
There's a comment about that in the `custom-destructor-types.h` header. I use to have a hard error if rtti wasn't enabled (in the header) but that doesn't work on windows, so now it's just a comment. I'll put a similar comment here. 

Basically, the issue is with rtti we get linker errors for destroying objects if we don't link against libc++.

Reply:
Thanks for repeating the comment. Each file will likely live its own life, and figuring out a couple of years from now why there is an `-fno-rtti` in one of the files but not in others would be difficult if we didn't have local comments.